CafeRecords/Pages/Index.cshtml

```diff
@@ -1,32 +1,31 @@
 ﻿@page
 @model IndexModel
 @{
-    ViewData["Title"] = "Home page";
-    CafeRecord.Cafe[] allCafes = (CafeRecord.Cafe[])ViewData["allCafes"];
```

Comment:
Removed the redundant lines of code as the mentioned ViewData Object is not used for displaying in html page.